lib/Remote/RemoteWebElement.php

```diff
@@ -371,6 +388,18 @@ public function setFileDetector(FileDetector $detector)
      */
     public function submit()
     {
+        if ($this->w3cCompliant) {
+            $this->executor->execute(DriverCommand::EXECUTE_SCRIPT, [
```

Comment:
I guess this will face this issue https://github.com/SeleniumHQ/selenium/issues/3398 in some forms:

>  If a form control (such as a submit button) has a name or id of submit it will mask the form's submit method.

See referenced issue for more universal way (using prototype method).

Reply:
I already use the prototype method. Am I missing something?